src/components/containers/TaskList.js

```diff
@@ -7,105 +7,147 @@ import { TaskSection } from '../presentational/Dropdown';
 };
 
-const categories = ['School', 'Work', 'Personal', 'Holidays'];
+const categories = ['School', 'Work', 'Personal', 'Holidays']; // temporary
 
 const TaskList = () => {
-  // EXAMPLE CREATE AND RETRIEVE TASK
-  let time_now = new Date().toISOString()
-  TaskService.postTask("Homework", "Alex", 1300, time_now, function (task) {
-    console.log("Task Posted")
-    console.log(task)
-
-    TaskService.getTask(1, function (task) {
-      console.log("Task Retrieved")
-      console.log(task)
-    })
-  })
-
-  // TaskService.getTask(1, function (task) {
-  //   console.log("Task Retrieved")
-  //   console.log(task)
-  // })
-
-  const [tasks, setTasks] = useState(initTasks);
-  const [schedTasks, setSchedTasks] = useState(scheduledTasks);
+  /* Sample TaskService use: */
+  /* let time_now = new Date().toISOString();
+  TaskService.postTask('Homework', 'Alex', 1300, function(task) {
+    console.log('Task Posted');
+    console.log(task);
+
+    TaskService.getTask(1, function(task) {
+      console.log('Task Retrieved');
+      console.log(task);
+    });
+  }); */
+
+  const [tasks, setTasks] = useState([]);
+  const [unscheduled, setUnscheduled] = useState([]);
+  const [scheduled, setScheduled] = useState([]);
+
   const [creating, setCreating] = useState(false);
   const [selected, setSelected] = useState([]);
 
+  const splitTasks = () => {
+    let schedArr = [],
+      unschedArr = [];
+    tasks.forEach((task) => {
+      if (task.scheduled) {
+        schedArr.push(task);
+      } else {
+        unschedArr.push(task);
+      }
+    });
+    setUnscheduled(unschedArr);
+    setScheduled(schedArr);
+  };
+
+  useEffect(() => {
+    /* split into scheduled and unscheduled every time tasks updates */
+    splitTasks();
+  }, [tasks]);
+
+  const createTask = (task) => {
+    const { name, description, duration, due } = task;
+    TaskService.postTask(name, description, duration, due, (task) => {
+      console.log('Creating:', task);
+      setTasks(
+        tasks.concat({
+          /* temporarily filling out extra fields */
+          ...task,
+          created: '10/10/10',
+          category: 'School',
+          dueDate: 'Tue Dec 10 2019',
+          scheduledDate: 'Tues 3-4'
+        })
+      );
+      setCreating(false);
+    });
+  };
+
   const selectTask = (id) => {
-    if (selected.indexOf(id) === -1) setSelected(selected.concat([id]));
-    else setSelected(selected.filter((num) => num !== id));
+    if (selected.indexOf(id) === -1) {
+      setSelected(selected.concat([id]));
+    } else {
+      setSelected(selected.filter((num) => num !== id));
+    }
   };
 
   const deleteTask = (id) => setTasks(tasks.filter((task) => task.id !== id));
 
-  const completeTask = (id) =>
-    setSchedTasks(schedTasks.filter((task) => task.id !== id));
-
-  const deleteSchedTask = (id) =>
-    setSchedTasks(schedTasks.filter((task) => task.id !== id));
+  const completeTask = deleteTask; // temporary
```

Comment:
A new commit by Alvin actually created the set task complete function in the Task Service, rebase and you should have it

Reply:
Updated to use complete task API.